src/tribol/mesh/MfemData.cpp

```diff
@@ -0,0 +1,876 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include "tribol/mesh/MfemData.hpp"
+
+#include "axom/slic.hpp"
+
+namespace tribol
+{
+
+SubmeshLORTransfer::SubmeshLORTransfer(
+  mfem::ParFiniteElementSpace& ho_submesh_fes,
+  mfem::ParMesh& lor_submesh
+)
+: lor_gridfn_ { CreateLORGridFunction(
+    lor_submesh,
+    std::make_unique<mfem::H1_FECollection>(1, lor_submesh.SpaceDimension()),
+    ho_submesh_fes.GetVDim()
+  ) },
+  lor_xfer_ { ho_submesh_fes, *lor_gridfn_.ParFESpace() }
+{
+  SLIC_WARNING_ROOT("LOR support is experimental at this time.");
+}
+
+void SubmeshLORTransfer::TransferToLORGridFn(
```

Comment:
I like other lexicon for the transfer function names. Let's modify so we are very clear we are transferring "from" and "to". Is this TransferHOToLORGridFn?